twilight-model/src/application/interaction/mod.rs

```diff
@@ -533,7 +533,7 @@ mod tests {
             user: None,
         };
 
-        serde_test::assert_tokens(
+        serde_test::assert_ser_tokens(
```

Comment:
Could you add a todo comment: "todo: switch to `assert_tokens`, see #2190" here?